mslib/msui/mscolab.py

```diff
@@ -1262,6 +1276,8 @@ def set_active_pid(self, item):
             # set new waypoints model to open views
             for window in self.ui.get_active_views():
                 window.setFlightTrackModel(self.waypoints_model)
+                if self.access_level == "viewer":
```

Comment:
Isn't there an else missing here?
Suppose you have a TopView open as a viewer, and change to another operation where you are a collaborator, the navbar does not enable itself again.

Reply:
good catch :)